### ISSUE TITLE
Grade: Needs Improvement

### ISSUE THREAD

**Issue by Rodrigap** [kirshr/bwa-u4-feature-request#1]
This is a really good start to your feature request! I appreciate that you tried to implement 2 features, but at times it makes the paper a little confusing because you bounce back and forth between the two. If you wanted to do two, I'd recommend separating them and doing them on different pages or documents. I really like both of these feature ideas! Adding a key press cue to search definitely makes your application much more intuitive and quicker to use, which will aid the user experience. Likewise, adding an audio player for users to preview tracks is really smart! I think you could have communicated these ideas a little more clearly. At first, it was unclear if these two features were supposed to be tied together (like if you press enter, you'd hear a preview of the song). You could also have put a little bit more into the justification behind your ideas. Your technical design for the enter key feature is very good! It's concise and I like that you reuse the current search method and simply add a keycode listener to cue it. Your technical design for the music player is a less clear. Your explanation is a little bit lacking in terms of the steps you'd take to implement it. Will you need to create a new component? What information will you need to feed the audio player for it to play the correct song? Try thinking about the concrete steps that you would need to take the implement this feature. Overall, you have very good feature ideas, but I'd encourage you to think more about the technical designs!

**Comment by kirshr** [comment]
I agree 100%. The original idea was to create a small audio player underneath each search result that would reach out to the spotify API, get the track and play it, all inside of the app. However, I could not get it to work at all, and had to settle for a less refined method. I will keep on trying until I get it!.

Thanks!
